components/speedreader/rust/lib/Cargo.toml

```diff
@@ -18,15 +18,15 @@ include = [
 ]
 
 [dependencies]
+adblock = "0.2.11"
+flate2 = { version = "1.0.20", features = ["rust_backend"], default-features = false }
 html5ever = "0.25.1"
-markup5ever_rcdom = "0.1.0"
-regex = "1"
+kuchiki = { git = "https://github.com/keur/kuchiki", branch = "speedreader" }
```

Comment:
If the repo is going to be included in brave builds, should the repo be moved or forked in the brave GH org, and possibly reviewed by sec team?

Reply:
Good point. Wasn't sure what to do here. So here is the difference between my branch and the master branch (very minor) can be seen here: https://github.com/kuchiki-rs/kuchiki/compare/master...keur:speedreader. It exposes some fields that were public in `Rcdom` and adds two metadata fields to massively simplify this implementation. No logic changes to the algorithm or anything like that. 

The existing implementation relied on the DOM being introspectable by API callers. Especially in the classification pass. Doesn't seem like there's any way around it, unfortunately. I tried to send these up, but upstream said if we are going to be doing this much DOM customization, we might as well have our own fork. https://github.com/kuchiki-rs/kuchiki/pull/87#issuecomment-803269129. So I went forward with that. I plan to keep my changes on top of their DOM extremely minimal.

Reply:
Moving this to brave org is a good idea. Someone with perms just needs to fork https://github.com/kuchiki-rs/kuchiki/ and give me write access to that.

Reply:
Sounds good. I think asking in #security-discussion is a great idea, since they can fork it, and review it before hand if they think its necessary

Reply:
Moved to Brave org in latest commit.